Dfe.Academies.Academisation.Domain/ApplicationAggregate/Application.cs

```diff
@@ -55,8 +55,8 @@ public void SetIdsOnCreate(int applicationId, int contributorId)
 	public CommandResult Update(
 		ApplicationType applicationType,
 		ApplicationStatus applicationStatus,
-		Dictionary<int, ContributorDetails> contributors,
-		Dictionary<int, SchoolDetails> schools)
+		IEnumerable<KeyValuePair<int, ContributorDetails>> contributors,
+		IEnumerable<KeyValuePair<int, SchoolDetails>> schools)
```

Comment:
can you explain this change please?

Reply:
Oh yeah, this was a major bug. When adding new Schools or Contributors, they must be added with an id of zero. If you add more than one at a time, you get an exception because the dictionary key must be unique.